ingeniamotion/monitoring/base_monitoring.py

```diff
@@ -137,6 +138,10 @@ def map_registers(self, registers: List[Dict[str, Union[int, str, REG_DTYPE]]])
             if not isinstance(register, str):
                 raise TypeError("Register has to be a string")
             register_obj = self.mc.info.register_info(register, subnode, servo=self.servo)
+            if register_obj.cyclic != RegCyclicType.TX:
```

Comment:
I think that RegCyclicType.TXRX should also be mappable

Reply:
Done [acff3ee](https://github.com/ingeniamc/ingeniamotion/pull/201/commits/acff3ee58b002b0ef913fd10ec8e528f0970c124)